provisioning/service/tests/TestAssert.cs

```diff
@@ -38,6 +38,11 @@ public static class TestAssert
             throw new AssertFailedException("{0}. Expected {1} exception but no exception is thrown".FormatInvariant(errorMessage, typeof(TException).ToString()));
         }
 
+        internal static void Throws<T>(Func<Task> p)
+        {
+            throw new NotImplementedException();
+        }
+
         public static TException Throws<TException>(Func<Task> action, string errorMessage = null) where TException : Exception
```

Comment:
We should switch to XUnit soon and avoid needing to have these extensions...

Reply:
This should be awaitable.

Reply:
We don't need it anymore, removed